ada/language/ast.py

```diff
@@ -9913,7 +9913,7 @@ def zip_with_params():
             ._.formal_part.abstract_formal_params,
 
             lambda c=T.DiscriminantConstraint:
-            c.subtype.cast(T.TypeDecl)._.discriminants_list,
```

Comment:
Not sure what this cast was for, I hope it won't break anything (it doesn't in our testsuite). Maybe you have an idea @raph-amiard ?

Reply:
1. I'm pretty sure the cast was wrong.
2. If it doesn't break anything in the testsuites, then I guess we'll see :)